toolkit/javascripts/clear-filters.js

```diff
@@ -23,6 +23,7 @@
     this.$clearFilterLink.on('click', function(event) {
         event.preventDefault();
         _this.clearFilters();
+        $(this).blur();
```

Comment:
no - it remains in the same position, ie. when you press tab it will go to the next element after the "Clear filters" button.